scenes/game_elements/props/decoration/hiding_mushroom/hiding_mushroom.tscn

```diff
@@ -140,5 +142,13 @@ collision_layer = 4
 [node name="CollisionShape2D" type="CollisionShape2D" parent="PlayerDetector"]
 shape = SubResource("CircleShape2D_v208a")
 
+[node name="AudioStreamPlayer2D_Hide" type="AudioStreamPlayer2D" parent="."]
```

Comment:
Both sound effects should have the "Bus" property set to "SFX" (an abbreviation of "sound effects"). Currently they are assigned to the "Master" bus, which means that the in-game volume control for sound effects doesn't affect their volume.

To test if this is set correctly, run the game, press Escape or Pause on your keyboard or Start/Menu on a controller, go to the options, and adjust the Sound Effects and Music volumes both to zero. Then walk past some mushrooms - they should be silenced.